doc/build_apps/logging_cpp.rst

```diff
@@ -200,4 +200,29 @@ Historical state always contains a receipt. Users wishing to implement a receipt
     :language: cpp
     :start-after: SNIPPET_START: get_historical_with_receipt
     :end-before: SNIPPET_END: get_historical_with_receipt
-    :dedent:
\ No newline at end of file
+    :dedent:
+
+User-defined claims in receipts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A user wanting to tie transaction-specific values to a receipt can do so by attaching a claims digest to their transaction:
```

Comment:
```suggestion
A user wanting to tie request/transaction-specific values to a receipt can do so by attaching a claims digest to their transaction:
```

Reply:
I think this is "transaction" in the broader sense, including the response, the request, and anything read/written in the KV. Anything the transaction handler has access to qualifies.